lib/elixir/lib/inspect/algebra.ex

```diff
@@ -250,6 +250,15 @@ defmodule Inspect.Algebra do
 
   # Elixir + Inspect.Opts conveniences
 
+  def get_inspect_fun() do
```

Comment:
let me know if we want to move forward with this approach and then I'll add docs etc.